packages/eyes/src/index.ts

```diff
@@ -131,11 +105,33 @@ async function waitForTestsCompletion(page: puppeteer.Page, url: string):
   return numTestsFailed;
 }
 
-export async function eyesTest(entry: string | string[]) {
+export async function eyesTest(entry: string | string[], apiKey: string, projectPath: string, webpackConfigPath: string) {
   let server: IServer | null = null;
+  const projectName = require(path.join(projectPath, 'package.json')).name;
```

Comment:
This line can throw an exception, and since this is an async function Node will terminate with status 0, as if the tests have passed successfully.

I suggest that you move the beginning of that try block on line 132 to the beginning of the function, and remove process.exit in apiKey and projectName validations.